src/stores/api/http/context.go

```diff
@@ -1,4 +1,4 @@
-package handlers
```

Comment:
If you don't mind you should apply same rename change to `aliases` package for better consistency

(I added the comment here because I didn't find better place to do so)